mcstatus/scripts/mcstatus.py

```diff
@@ -68,7 +68,57 @@ def status():
             ] if response.players.sample is not None else "No players online"
         )
     )
-
+    
+@cli.command(short_help="all available server information in json")
+def json():
+    """
+    Prints server status and query in json. Supported by all Minecraft
+    servers that are version 1.7 or higher.
+    """
+    try:
+        ping = server.ping()
+        online=True
+    except Exception:
+        online=False
+    output = '{'
+    if online:
+        response1 = server.status(retries=1)
+        try:
+            response2 = server.query(retries=1)
+            query = True
+        except Exception:
+            query = False
+        output+='"online":true,'
```

Comment:
I like the idea of json! But instead of adding a little bit at a time to a string variable, would you mind converting this to using python's built-in json library?  https://docs.python.org/2/library/json.html
`json.dumps` would probably work great here, and make the code a bit easier to read.